<?xml version="1.0"?>
<launch>
  <param name="quad_frame" value="NWU" />
  <node pkg="atl_ros" name="atl_imu" type="imu_node" output="screen" launch-prefix="sudo -E" required="true" />
</launch>
